app/javascript/tests/public_health/query/AdvancedFilter.test.js

```diff
@@ -0,0 +1,1040 @@
+    expect(wrapper.find('.advanced-filter-additional-filter-options').prop('value')).toEqual(mockFilterAdditionalOption.additionalFilterOption);
+    expect(wrapper.find('.advanced-filter-number-options').prop('value')).toEqual(mockFilterAdditionalOption.numberOption);
+    expect(wrapper.find('.advanced-filter-number-input').prop('value')).toEqual(mockFilterAdditionalOption.value);
+  });
+
+  it('Relative date custom tooltip dynamically updates as options change', () => {
+    const wrapper = getWrapper();
+    wrapper.find(Button).simulate('click');
+    wrapper.find('.advanced-filter-select').simulate('change', { value: mockFilterDefaultCustomRelativeOption.filterOption.name });
+    expect(wrapper.find(ReactTooltip).exists()).toBeFalsy();
+    wrapper.find('.advanced-filter-relative-options').simulate('change', { target: { value: 'custom' } });
+    expect(wrapper.find(ReactTooltip).exists()).toBeTruthy();
+    expect(wrapper.find(ReactTooltip).find('span').text()).toEqual(`Latest Report “past” relative date periods include records dated through today’s date. The current setting of "past 1 days" will return records with Latest Report date from ${moment(new Date()).subtract(1,'d').format('MM/DD/YY')} through now.`);
+    wrapper.find('.advanced-filter-number-input').simulate('change', { target: { value: 3 } });
+    expect(wrapper.find(ReactTooltip).find('span').text()).toEqual(`Latest Report “past” relative date periods include records dated through today’s date. The current setting of "past 3 days" will return records with Latest Report date from ${moment(new Date()).subtract(3,'d').format('MM/DD/YY')} through now.`);
+    wrapper.find('.advanced-filter-unit-input').simulate('change', { target: { value: 'weeks' } });
+    expect(wrapper.find(ReactTooltip).find('span').text()).toEqual(`Latest Report “past” relative date periods include records dated through today’s date. The current setting of "past 3 weeks" will return records with Latest Report date from ${moment(new Date()).subtract(3,'weeks').format('MM/DD/YY')} through now.`);
+    wrapper.find('.advanced-filter-when-input').simulate('change', { target: { value: 'next' } });
+    expect(wrapper.find(ReactTooltip).find('span').text()).toEqual(`Latest Report “next” relative date periods include records with today’s date as of the current time. The current setting of "next 3 weeks" will return records with Latest Report date from now through ${moment(new Date()).add(3,'weeks').format('MM/DD/YY')}.`);
+    wrapper.find('.advanced-filter-number-input').simulate('change', { target: { value: 1 } });
+    expect(wrapper.find(ReactTooltip).find('span').text()).toEqual(`Latest Report “next” relative date periods include records with today’s date as of the current time. The current setting of "next 1 weeks" will return records with Latest Report date from now through ${moment(new Date()).add(1,'weeks').format('MM/DD/YY')}.`);
+  });
+
+  // TO DO: between option bound prevention
```

Comment:
Is this complete? If so, let's remove.

Reply:
no it is not - left this in there so I would remember to write it as a part of [SARAALERT-1242](https://tracker.codev.mitre.org/browse/SARAALERT-1242) for whatever solution the PO team thinks is best

Reply:
Let's remove for now and if you want you can add the reminder as part of that ticket.

Reply:
Done and added a bullet to the definition of done for that ticket